packages/block-library/src/navigation/style.scss

```diff
@@ -308,8 +301,9 @@
 
 // Default background and font color.
 .wp-block-navigation:not(.has-background) {
+	// @todo: check this one separately.
 	.submenu-container, // This target items created by the Page List block.
-	.wp-block-navigation__container .wp-block-navigation-link__container {
+	.wp-block-navigation__container .wp-block-navigation__submenu-container {
```

Comment:
We should be able to remove `submenu-container` here, and use just `.wp-block-navigation__submenu-container` to target all submenus. I'm not sure why the `.wp-block-navigation__container` was added in the first place; it's not needed to target only the submenus, because the submenu classname does that already, so it may have been added to increase specificity. But if we do need additional specificity, we can just duplicate `.wp-block-navigation__submenu-container`.